src/lib/Scenes/Home/Components/Sales/index.tsx

```diff
@@ -21,7 +21,7 @@ class Sales extends React.Component<Props, State> {
   }
 
   get data() {
-    const { sales } = this.props.query
+    const sales = this.props.query.salesConnection.edges.map(({ node }) => node)
```

Comment:
Any chance `edges` or `salesConnection` here could be `null`?

Reply:
`salesConnection` can’t be, `edges` might but I expect things to then first blow up in MP.